### PR TITLE
cpu and gpu Dockerfiles for ppc64le

### PR DESCRIPTION
**Pull request by wdirons**
Adding Dockerfile.cpu.ppc64le and Dockerfile.gpu.ppc64le to enable the ability
to do builds using docker on ppc64le. Also enables the ability to run
ci_sanity.sh (from ci_build.sh) on ppc64le.

Modified ci_build.sh and ci_parameterized_build.sh to accept container types
that start with cpu or gpu.

Added install_bazel_from_source.sh and install_buildifier_from_source.sh install
scripts to avoid installing x86 versions of the binaries. These scripts could be
used by other platforms in the future.

### CODE REVIEW

File: tensorflow/tools/ci_build/install/install_golang_ppc64el.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```
Comment (gunan) [inline tensorflow/tools/ci_build/install/install_golang_ppc64el.sh]: Minor nit.
filename, should it be *ppc64le.sh

Comment (wdirons) [reply tensorflow/tools/ci_build/install/install_golang_ppc64el.sh]: Thanks for the comment, I fixed the filename and it's invocation from the two dockerfiles.

**Review by gunan — APPROVED** [review]

File: tensorflow/tools/ci_build/ci_parameterized_build.sh [hunk 2]
```diff
@@ -418,12 +418,12 @@ if [[ ${TF_BUILD_IS_PIP} == "no_pip" ]] ||
     BAZEL_TARGET=${TF_BUILD_BAZEL_TARGET}
   fi
 
-  if [[ ${CTYPE} == "cpu" ]] || \
+  if [[ ${CTYPE} == cpu* ]] || \
```
Comment (wdirons) [inline tensorflow/tools/ci_build/ci_parameterized_build.sh]: I wasn't sure if I should wildcard cpu and gpu in these two files or if I should explicitly list cpu.ppc64le as an option like

```
 if [[ ${CTYPE} == "cpu" ]] || \
    [[ ${CTYPE} == "cpu.ppc64le" ]]; then \
    [[ ${CTYPE} == "debian.jessie.cpu" ]]; then
```

Comment (gunan) [reply tensorflow/tools/ci_build/ci_parameterized_build.sh]: To be honest, this file is in the works of being replaced. So either is OK with me.

**Review by gunan — APPROVED** [review]

**Pull request merged**